src/components/ConnectWallet/Wallets/ElectrumDesktop.vue

```diff
@@ -1,35 +1,31 @@
 <template>
-  <connection-details name="Electrum (Desktop)">
+  <connection-details name="Electrum Wallet (Desktop)">
     <step-list>
       <step>
         Make sure Tor is up and running on your system.
         <b-link v-b-modal.tor-modal>Click here</b-link> for instructions.
       </step>
       <step>
-        Open Electrum and click on the colored dot at the bottom right to open server setttings.
+        Open Electrum Wallet and click the colored dot on the bottom right to open network settings.
       </step>
       <step>
-        Disable <span class="font-weight-bold">"Select server automatically"</span>.
+        Uncheck <span class="font-weight-bold">"Select server automatically"</span>.
       </step>
       <step>
         In the <span class="font-weight-bold">"Server"</span>, enter
-        <input-copy :value="urls.bitcoin.electrum.address" auto-width></input-copy>
-        <input-copy :value="urls.bitcoin.electrum.port" auto-width></input-copy>
+        <input-copy :value="`${urls.bitcoin.electrum.address}:${urls.bitcoin.electrum.port}:t`" auto-width></input-copy>
       </step>
       <step>
         Go to the <span class="font-weight-bold">"Proxy"</span> tab.
       </step>
       <step>
         Check <span class="font-weight-bold">"Use Tor proxy at port 9050"</span>
-        or manually enter this:
-        <input-copy value="127.0.0.1" auto-width></input-copy>
-        <input-copy value="9050" auto-width></input-copy>
```

Comment:
I think we should let these instructions as I've seen cases where Electrum doesn't detect Tor running on port 9050.

Reply:
Are you sure about that? I thought it always provided that option if it found Tor on port 9050 or 9150. And the only time it didn't show the option was when it was unable to find Tor...

Reply:
If you say so 👌